components/usage/pkg/db/workspace_instance_test.go

```diff
@@ -191,3 +191,29 @@ func TestListWorkspaceInstancesInRange(t *testing.T) {
 
 	require.Len(t, retrieved, len(valid))
 }
+
+func TestListWorkspaceInstancesInRange_InBatches(t *testing.T) {
+	conn := db.ConnectForTests(t)
+
+	workspaceID := "gitpodio-gitpod-gyjr82jkfnd"
+	var instances []db.WorkspaceInstance
+	for i := 0; i < 1100; i++ {
+		instances = append(instances, dbtest.NewWorkspaceInstance(t, db.WorkspaceInstance{
+			ID:           uuid.New(),
+			WorkspaceID:  workspaceID,
+			CreationTime: db.NewVarcharTime(time.Date(2022, 05, 15, 12, 00, 00, 00, time.UTC)),
+			StartedTime:  db.NewVarcharTime(time.Date(2022, 05, 15, 12, 00, 00, 00, time.UTC)),
+			StoppedTime:  db.NewVarcharTime(time.Date(2022, 05, 15, 13, 00, 00, 00, time.UTC)),
+		}))
+
+	}
+
+	tx := conn.CreateInBatches(&instances, 1000)
```

Comment:
Why do we need to `CreateInBatches` here?

Reply:
We don't strictly, but felt like I should start a pattern of not doing things with more than 1k entries outside batches